ietf-wdm-interface.tree

```diff
@@ -116,14 +122,14 @@ module: ietf-ext-xponder-wdm-if
           +--ro pre-fec-ber?                decimal64
 
   notifications:
-    +---n opt-if-och-central-frequency-change
+    +---n wdm-if-central-frequency-change
     |  +--ro if-name?
     |  |       -> /if:interfaces/interface/name
-    |  +--ro new-opt-if-och-central-frequency
+    |  +--ro new-wdm-if-central-frequency
     |     +--ro central-frequency?   l0-types:frequency-thz
-    +---n opt-if-och-mode-change
+    +---n wdm-if-mode-change
     |  +--ro if-name?   -> /if:interfaces/interface/name
     |  +--ro mode-id?   string
```

Comment:
Do we need to define these notifications? I think they can be notified using standard Netconf/Restconf mechanisms